src/platform_impl/ios/view.rs

```diff
@@ -508,9 +508,7 @@ pub unsafe fn create_window(
             let () = msg_send![uiscreen, setCurrentMode: video_mode.video_mode.screen_mode];
             msg_send![window, setScreen:video_mode.monitor().ui_screen()]
         }
-        Some(Fullscreen::Borderless(ref monitor)) => {
```

Comment:
It seems you're not using the latest version of rustc/rustfmt. This change needs to be reverted to pass `cargo fmt --check`.

Reply:
It seems like --check is an invalid argument.

Reply:
Seems like its fixed now